client/data/account/actions.js

```diff
@@ -40,11 +40,18 @@ export function* refreshAccount() {
 
 		// Check new payment methods available for account.
 		const newPaymentMethods = activeCapabilitiesAfterRefresh.filter(
-			( paymentMethod ) =>
-				! activeCapabilitiesBeforeRefresh.includes( paymentMethod ) &&
-				PaymentMethodsMap[
-					paymentMethod.replace( '_payments', '' )
-				] !== undefined
+			( capability ) => {
+				const paymentMethodFromCapability =
+					capability === 'us_bank_account_ach_payments'
+						? 'us_bank_account'
```

Comment:
maybe we should use the `PAYMENT_METHOD_ACH` constant defined here? https://github.com/woocommerce/woocommerce-gateway-stripe/blob/c9d4bb6082833048ff344fb222a27de138a8bfa6/client/stripe-utils/constants.js#L25

Reply:
True, I was considering that, but was worried about the import just in case. I'll treat this as a fast follow enhancement though.

Reply:
sounds good!